google-cloud-pubsub/pom.xml

```diff
@@ -54,6 +54,36 @@
       <version>${grpc.version}</version>
     </dependency>
     <dependency>
+      <groupId>com.google.guava</groupId>
+      <artifactId>guava</artifactId>
+      <version>20.0</version>
```

Comment:
I think this was updated earlier to get stats package which we don't use right now. Reverted.